lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -14,6 +14,11 @@
 except ImportError as err:
     raise ImportError("The GTK3 backends require PyGObject") from err
 
+try:
+    gi.require_version
+except AttributeError as err:
+    raise ImportError("The GTK3 backends require PyGObject") from err
```

Comment:
Does the error message above need to be corrected then?  I do not fully understand how PyGObject / gobject introspection splits itself up in either and upstream sense or the downstream packaging sense (which I assume different packagers make different choices?).

"No, this is the right error message in both cases" is perfectly good answer, but if we can make this a bit more precise that would be good.

Reply:
> Does the error message above need to be corrected then?

I think it's the correct error as the remediation is (on Arch Linux) to install `python-gobject`. `import gi` doesn't fail because other packages populate the /usr/lib/python3.13/site-packages/gi/ path, e.g. with /usr/lib/python3.13/site-packages/gi/overrides/Atspi.py.